WordPressKit/WordPressAPIError.swift

```diff
@@ -21,15 +21,15 @@ public enum WordPressAPIError<EndpointError>: Error where EndpointError: Localiz
     case unknown(underlyingError: Error)
 }
 
-extension WordPressComOAuthError: LocalizedError {
+extension WordPressAPIError: LocalizedError {
 
     public var errorDescription: String? {
         switch self {
         case .requestEncodingFailure, .unparsableResponse:
             // These are usually programming errors.
             return Self.unknownErrorMessage
         case let .endpointError(error):
-            return error.errorDescription
+            return error.errorDescription ?? Self.unknownErrorMessage
```

Comment:
> The implementation should always return a non-nil error message

Gotcha. So even though `errorDescription` at the `LocalizedError` level allows for a `nil` value, we agree not to do that.

Would it be worth documenting it?

Reply:
_Maybe..._ there could be an internal `_errorDescription` or `strongErrorDescription` or some other better name, that is `String` and `public var errorDescription: String?` could delegate to that. The idea being to have the compiler help us remember not to return any `Optional.none`.

Reply:
Good point. I didn't make more changes to the implementation because I thought the returning nil case is covered by the added unit tests. But I'll add a bit documentation too.